packages/devtools-modules/client/shared/components/splitter/SplitBox.js

```diff
@@ -60,8 +60,9 @@ const SplitBox = React.createClass({
   getInitialState() {
     return {
       vert: this.props.vert,
-      width: this.props.initialWidth || this.props.initialSize,
-      height: this.props.initialHeight || this.props.initialSize
+      // We user integers for these properties
+      width: parseInt(this.props.initialWidth || this.props.initialSize),
+      height: parseInt(this.props.initialHeight || this.props.initialSize)
```

Comment:
Why that ? parseInt always defaults to 10 these days, in all current browsers. Forcing the radix is when you need to support very old versions, I doubt this is our case.